README.md

```diff
@@ -1,3 +1,4 @@
+[comment]: <> (//@todo: Should we add something about the new Mojo?)
```

Comment:
Of course

Reply:
Ale, are we going to merge this PR first and then create another for the new Mojo?

If so, I think that addition to the README should be part of the PR where the new Mojo gets introduced